docs/culture/psychological-safety.md

```diff
@@ -0,0 +1,107 @@
+
+In order for collaboration to occur we must be able to share information, ask questions and express our opinions openly {cite:p}`edmondson_psychological_2023`. When we ask these questions of people outside of our disciplinary boundary, we might be stepping into situations were we feel incompetent or intrusive. This is where creating a psychologically safe environment can help.
+
+:::{admonition} Psychological Safety: Defined
+:class: note
+Psychological safety is a perception related to the consequences of taking *interpersonal risks* {cite:p}`edmondson_psychological_1999`. When people feel psychologically safe they can "express themselves physically, cognitively and emotionally" rather than disengage or "withdraw and defend their personal selves" {cite:p}`kahn_psychological_1990`.
+:::
+
+In this lesson we will work on developing more insight into risk and all the ways it might show up in a short format training setting. We'll do some exercises to assess our individual sense of risk, and build our capacity to assess how others might be experiencing risk in a classroom, small group and 1x1 learning setting. Then we will share strategies for increasing psychological safety by highlighting the positive outcomes that can occur when we willingly take risks.
+
+:::{admonition} Note 
+:class: warning
+Throughout this lesson we will be speaking about creating conditions for risk-taking in the context of interpersonal exchanges in a learning setting. It is important to separate this from other forms of risk that might impact people's physical or emotional wellbeing. We will never advocate for risks that bring about harm to others. For more information please visit our [Code of Conduct](https://escience.washington.edu/about/code-of-conduct/).
+:::
+
+## Hackweeks: a high risk environment?
+
+Let's begin by thinking about the factors that influence our perception of risk at one of our events. Over the years we as organizers have often noticed participants feeling anxious and wondering if they belong at a hackweek. In some ways this is understandable! Here are some reasons why:
+
+* Hackweeks are a relatively new modality of learning and participants are often not sure what they are signing up for. 
+* Hackweeks actively promote networking with people you haven't met before, which might cause a sense of anxiety for some.
+* The short duration of hackweeks may cause many participants to feel a sense of urgency to get a lot done in the time provided. People might also feel they need to make the most of the money they invested in attending the event in person.
+* Hackweeks require participants to make quick decisions about tutorials they attend and projects they join, often with limited time for deliberation.
```

Comment:
Thoughts about adding a point addressing career/experience stages. E.g.: Hackweeks bring together novices and experts from across disciplines and career stages. It can be easy to for participants to notice where they are novices and harder to notice where they are experts.